cmd/run.php

```diff
@@ -24,11 +24,19 @@
 $option = $climate->radio("Choose data to fetch:", [
     GithubScrapper::LOCATION => "organization location",
     GithubScrapper::URL => "repository URL",
+    GithubScrapper::WEB => "website",
+    GithubScrapper::TWITTER => "twitter",
+    GithubScrapper::EMAIL => "email",
+    GithubScrapper::VERIFICATION => "verification",
```

Comment:
When these consts will be as cases in backed enum class, then you will be able to get all values by enum method
```php
public static function values(): array
    {
       return array_column(self::cases(), "value");
    }
```
https://www.php.net/manual/en/language.enumerations.static-methods.php#126866